src/views/Proposal/Votes.vue

```diff
@@ -0,0 +1,218 @@
+
+  loadingMore.value = true;
+  const newVotes = await network.value.api.loadProposalVotes(
+    props.proposal,
+    {
+      limit: LIMIT,
+      skip: votes.value.length
+    },
+    choiceFilter.value,
+    sortBy.value
+  );
+  hasMore.value = newVotes.length === LIMIT;
+  votes.value = [...votes.value, ...newVotes];
+  loadingMore.value = false;
+}
+
+onMounted(() => {
+  loadVotes();
+});
+
+watch(
+  () => props.proposal.id,
+  (toId, fromId) => {
+    if (toId === fromId) return;
+
+    reset();
+  }
+);
+
+watch([sortBy, choiceFilter], () => {
+  reset();
+  loadVotes();
+});
+</script>
+
+<template>
+  <div class="flex mx-4 my-4 gap-3 justify-between">
+    <UiSelect
+      v-model="choiceFilter"
+      title="Choice"
+      gap="12px"
+      placement="left"
+      :items="[
+        { key: 'any', label: 'Any' },
+        { key: 'for', label: 'For', indicator: 'bg-choice-for' },
+        { key: 'against', label: 'Against', indicator: 'bg-choice-against' },
+        { key: 'abstain', label: 'Abstain', indicator: 'bg-choice-abstain' }
+      ]"
+    />
+  </div>
+
+  <BlockInfiniteScroller :loading-more="loadingMore" @end-reached="handleEndReached">
+    <table class="text-right w-full table-fixed">
+      <colgroup>
+        <col class="w-[50%] lg:w-[40%]" />
+        <col class="w-[25%] lg:w-[20%]" />
+        <col class="w-[25%] lg:w-[20%]" />
+        <col class="w-[60px] lg:w-[20%]" />
+        <col class="w-[0px] lg:w-[100px]" />
+      </colgroup>
+      <thead>
+        <tr class="border-b">
+          <th class="text-left pl-4 eyebrow text-skin-text pb-2">Votes</th>
+          <th class="eyebrow text-skin-text">
+            <div
+              class="flex justify-end items-center"
+              :class="{
+                'text-skin-link': sortBy === 'created-desc' || sortBy === 'created-asc'
+              }"
+              @click="handleSortChange('created')"
+            >
+              <span>Date</span>
+              <IH-arrow-sm-down v-if="sortBy === 'created-desc'" />
+              <IH-arrow-sm-up v-else />
+            </div>
+          </th>
+          <th class="eyebrow text-skin-text hidden lg:table-cell">Choice</th>
+          <th class="eyebrow text-skin-text">
+            <div
+              class="flex justify-end items-center"
+              :class="{
+                'text-skin-link': sortBy === 'vp-desc' || sortBy === 'vp-asc'
+              }"
+              @click="handleSortChange('vp')"
+            >
+              <span class="truncate pl-2">Voting Power</span>
+              <IH-arrow-sm-down v-if="sortBy === 'vp-desc'" />
+              <IH-arrow-sm-up v-else />
+            </div>
+          </th>
+          <th />
+        </tr>
+      </thead>
+      <td v-if="!loaded" colspan="5">
+        <UiLoading class="p-4 block text-center" />
+      </td>
+      <template v-else>
+        <div v-if="votes.length === 0" class="p-4 text-center col-span-full">
```

Comment:
The positioning of the message here seems off
<img width="1506" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/51686767/fd8c6d50-257d-4183-8742-516da64704e6">